src/server/rest/v1/service/CarService.ts

```diff
@@ -218,98 +217,76 @@ export default class CarService {
         module: MODULE_NAME, method: 'handleCreateCar'
       });
     }
-    // Check if component is active
-    UtilsService.assertComponentIsActiveFromToken(req.user, TenantComponents.CAR,
-      Action.CREATE, Entity.CAR, MODULE_NAME, 'handleCreateCar');
     // Filter
     const filteredRequest = CarSecurity.filterCarCreateRequest(req.body);
     // Check
     UtilsService.checkIfCarValid(filteredRequest, req);
-    // Check and get CarCatalog
-    const carCatalog = await UtilsService.checkAndGetCarCatalogAuthorization(
-      req.tenant, req.user, filteredRequest.carCatalogID, Action.READ, action, { withImage: true }, true);
-    UtilsService.assertObjectExists(action, carCatalog, `Car Catalog ID '${filteredRequest.carCatalogID}' does not exist`,
-      MODULE_NAME, 'handleCreateCar', req.user);
-    // Keep the current user to add for Basic role
-    const carUserToAdd = filteredRequest.usersAdded.find((carUser) => carUser.user.id === req.user.id);
-    // Check Car
-    const car = await CarStorage.getCarByVinLicensePlate(req.tenant,
-      filteredRequest.licensePlate, filteredRequest.vin, {
-        withUsers: Authorizations.isBasic(req.user) ? true : false,
+    // Only Admin can create Pool Car
+    if (filteredRequest.type === CarType.POOL_CAR && !Authorizations.isAdmin(req.user)) {
```

Comment:
I think this case should be handled by the authorizations framework

Reply:
I added new auth named 'CREATE_POOL_CAR' to Admin and Site Admin

Reply:
Finally I introduced the Assert authorizations to check the content of an Updated/Created object.